packages/form-js-viewer/src/render/components/form-fields/Table.js

```diff
@@ -263,11 +265,30 @@ Table.config = {
       ],
     };
   },
-  initialDemoData: [
-    { id: 1, name: 'John Doe', date: '31.01.2023' },
-    { id: 2, name: 'Erika Muller', date: '20.02.2023' },
-    { id: 3, name: 'Dominic Leaf', date: '11.03.2023' }
-  ],
+
+  /**
+   *
+   * @param {Field} field
+   */
+  generateInitialDemoData: (field) => {
```

Comment:
Generally, let's be careful with these kinds of changes. Our config object is tied in with the custom component framework and should be treated more or less as public API. Of course we're never obligated but changing how it's structured is something we should not do often. 

I don't think in this case it matters that much, because the whole initialDemoData stuff is a very unique definition here, but I'm concerned people will want to hook into this and we end up getting rid of it in the future.

Maybe it's worth annotating this as experimental, so that people seeing this implementation don't try to copy and rely on it.

Reply:
Or moving it to an experimental sub-object, which should get the point across 🤔

Reply:
@Skaiir I added the `@experimental` tag